dev/NavigationView/NavigationView.cpp

```diff
@@ -340,17 +340,6 @@ void NavigationView::OnApplyTemplate()
         backButtonToolTip.Content(box_value(navigationBackButtonToolTip));
     }
 
-    if (auto buttonHolderGrid = GetTemplateChildT<winrt::Grid>(c_buttonHolderGrid, controlProtected))
-    {
-        // TrySetNewFocusedElement call in OnButtonHolderGridGettingFocus is RS4 only
-        if (buttonHolderGrid.try_as<winrt::IUIElement8>())
-        {
-            buttonHolderGrid.XYFocusKeyboardNavigation(winrt::XYFocusKeyboardNavigationMode::Enabled);
-            buttonHolderGrid.TabFocusNavigation(winrt::KeyboardNavigationMode::Once);       
-            m_buttonHolderGettingFocusRevoker = buttonHolderGrid.GettingFocus(winrt::auto_revoke, { this, &NavigationView::OnButtonHolderGridGettingFocus });
```

Comment:
Is m_buttonHolderGettingFocusRevoker still needed now?

Reply:
Yep, looks unnecessary. Will remove.